nautobot/dcim/models/device_components.py

```diff
@@ -598,6 +601,7 @@ def to_csv(self):
             self.mgmt_only,
             self.description,
             self.get_mode_display(),
+            self.get_status_display(),
         )
 
     def clean(self):
```

Comment:
Thinking about this from a not-breaking-existing-usage standpoint, would it make sense for `clean()` and/or `save()` to automatically set `self.status = status.objects.get(slug="active")` (or equivalent) if an Interface or VMInterface is instantiated without setting its status? Thinking about the impact to Jobs in particular here.